### ISSUE TITLE
是否可以支持zookeeper，实现和apisix-seed联动？

### ISSUE THREAD

**Issue by Bigwen-1** [apache/apisix-dashboard#2579]
# Feature request

## Please describe your feature

A clear and concise description of what you want and what your use case is.

**Comment by bzp2010** [comment]
Yes, we have plans to implement it, but I'm currently too busy with APISIX Dashboard v3 to finish it for now. It is actually quite simple and you can refer to the modifications in https://github.com/bzp2010/apisix-dashboard/tree/test-for-seed to implement it on your own branch and use the official version when a future update is released.

**Comment by bzp2010** [comment]
#2353 and #2354 were deemed unfeasible due to cost and lack of support.

Per the project scope and principles in #2981, any functionality that is not directly supported by the Admin API will not be implemented in the UI, if this is required then please start a discussion on APISIX about built-in zookeeper service discovery support.

If you have any questions, feel free to start a new issue!
